python/python/tests/test_async_client.py

```diff
@@ -757,6 +757,25 @@ async def test_hlen(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.hlen(key2)
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_hvals(self, redis_client: TRedisClient):
+        key = get_random_string(10)
+        key2 = get_random_string(5)
+        field = get_random_string(5)
+        field2 = get_random_string(5)
+        field_value_map = {field: "value", field2: "value2"}
+
+        assert await redis_client.hset(key, field_value_map) == 2
+        assert await redis_client.hvals(key) == ["value", "value2"]
+        assert await redis_client.hdel(key, [field]) == 1
+        assert await redis_client.hvals(key) == ["value2"]
+        assert await redis_client.hvals("non_existing_key") == []
+
+        assert await redis_client.set(key2, "value") == OK
```

Comment:
Please add a test where key exists but it is not a hash

Reply:
Lines 775-777

Reply:
Oh right, thanks